[2024-08-19T13:04:28.579+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-19T13:04:28.781+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: postgres_to_snowflake.get_max_id_estados manual__2024-08-19T13:03:58.191100+00:00 [queued]>
[2024-08-19T13:04:28.864+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: postgres_to_snowflake.get_max_id_estados manual__2024-08-19T13:03:58.191100+00:00 [queued]>
[2024-08-19T13:04:28.865+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-08-19T13:04:29.024+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): get_max_id_estados> on 2024-08-19 13:03:58.191100+00:00
[2024-08-19T13:04:29.255+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:62: DeprecationWarning: This process (pid=55904) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-08-19T13:04:29.284+0000] {standard_task_runner.py:64} INFO - Started process 55923 to run task
[2024-08-19T13:04:30.969+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'postgres_to_snowflake', 'get_max_id_estados', 'manual__2024-08-19T13:03:58.191100+00:00', '--job-id', '162', '--raw', '--subdir', 'DAGS_FOLDER/ingestion.py', '--cfg-path', '/tmp/tmpntbt3i76']
[2024-08-19T13:04:31.473+0000] {standard_task_runner.py:91} INFO - Job 162: Subtask get_max_id_estados
[2024-08-19T13:04:32.762+0000] {task_command.py:426} INFO - Running <TaskInstance: postgres_to_snowflake.get_max_id_estados manual__2024-08-19T13:03:58.191100+00:00 [running]> on host ff16473fb2e3
[2024-08-19T13:04:35.059+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='postgres_to_snowflake' AIRFLOW_CTX_TASK_ID='get_max_id_estados' AIRFLOW_CTX_EXECUTION_DATE='2024-08-19T13:03:58.191100+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-08-19T13:03:58.191100+00:00'
[2024-08-19T13:04:35.065+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-19T13:04:35.221+0000] {base.py:84} INFO - Using connection ID 'snowflake' for task execution.
[2024-08-19T13:04:35.271+0000] {connection.py:408} INFO - Snowflake Connector for Python Version: 3.11.0, Python Version: 3.12.4, Platform: Linux-5.15.153.1-microsoft-standard-WSL2-x86_64-with-glibc2.36
[2024-08-19T13:04:35.319+0000] {connection.py:1258} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2024-08-19T13:04:38.419+0000] {cursor.py:1149} INFO - Number of results in first chunk: 1
[2024-08-19T13:04:38.433+0000] {connection.py:773} INFO - closed
[2024-08-19T13:04:38.616+0000] {connection.py:779} INFO - No async queries seem to be running, deleting session
[2024-08-19T13:04:38.708+0000] {python.py:237} INFO - Done. Returned value was: 27
[2024-08-19T13:04:38.719+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-19T13:04:40.076+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=postgres_to_snowflake, task_id=get_max_id_estados, run_id=manual__2024-08-19T13:03:58.191100+00:00, execution_date=20240819T130358, start_date=20240819T130428, end_date=20240819T130440
[2024-08-19T13:04:40.313+0000] {local_task_job_runner.py:243} INFO - Task exited with return code 0
[2024-08-19T13:04:40.367+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
